zhaquirks/xiaomi/aqara/remote_h1.py

```diff
@@ -0,0 +1,228 @@
+"""Aqara H1-series wireless remote"""
+import zigpy.types as t
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import Basic, Identify, OnOff, PowerConfiguration
+from zhaquirks import PowerConfigurationCluster
+from zhaquirks.const import (
+    ALT_DOUBLE_PRESS,
+    ALT_SHORT_PRESS,
+    ARGS,
+    BUTTON,
+    COMMAND,
+    COMMAND_OFF,
+    COMMAND_TOGGLE,
+    DEVICE_TYPE,
+    DOUBLE_PRESS,
+    ENDPOINT_ID,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LEFT,
+    LONG_PRESS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    RIGHT,
+    SHORT_PRESS,
+    TRIPLE_PRESS,
+)
+from zhaquirks.xiaomi import (
+    LUMI,
+    BasicCluster,
+    XiaomiCustomDevice,
+    XiaomiAqaraE1Cluster,
+)
+from zhaquirks.xiaomi.aqara.opple_remote import (
+    MultistateInputCluster,
```

Comment:
`MultistateInputCluster` has been implemented multiple times in different locations, including `remote_b186acn01` `remote_b286acn01` `opple_remote`, each with slight difference. Maybe it's a good idea to re-organize them?